### PR TITLE
client/core&client/webserver: Show number of expired unrefunded bond on dex settings view

### LINKED ISSUE #2625: Display info about expired unrefunded bonds
This would be info about the bonds in `dexAccount.expiredBonds` or `dexAcctBondState.expiredBonds`. We don't even have a field for it in `core.ExchangeAuth` now that I'm looking. We should display the number of expired bonds in the table on dexsettings.

image

### PR DESCRIPTION
**Pull request by ukane-philemon**
Closes #2625

Screenshot 2023-12-04 at 7 28 14 PM

### CODE REVIEW

File: client/core/types.go [hunk 1]
```diff
@@ -676,6 +676,8 @@ type ExchangeAuth struct {
 	PenaltyComps uint16 `json:"penaltyComps"`
 	// PendingBonds are currently pending bonds and their confirmation count.
 	PendingBonds []*PendingBondState `json:"pendingBonds"`
+	// expiredBondsPendingRefund is the number of expired but unrefunded bonds.
+	ExpiredBondsPendingRefund int64 `json:"expiredBondsPendingRefund"`
```
Comment (buck54321) [inline client/core/types.go:680]: Doc capitalization does not match.

File: client/webserver/locales/en-us.go [hunk 2]
```diff
@@ -466,35 +466,36 @@ var EnUS = map[string]string{
...
+	"Current Price":                "Current Price",
+	"Available Balance":            "Available Balance",
+	"ticket_qty_label":             "How Many?",
+	"Ticket History":               "Ticket History",
+	"No tickets to show":           "No tickets to show",
+	"Treasury Spends":              "Treasury Spends",
+	"No":                           "No",
+	"Yes":                          "Yes",
+	"Treasury Keys":                "Treasury Keys",
+	"expired_bonds_pending_refund": "Expired Bonds Pending Refund",
```
Comment (buck54321) [inline client/webserver/locales/en-us.go:500]: Pick a shorter key please.

Comment (buck54321) [inline client/webserver/locales/en-us.go:500]: Maybe just "Bonds Pending Refund" for the value?

**Review by buck54321 — CHANGES_REQUESTED** [review]

File: client/webserver/locales/en-us.go (hunk 2 above)
Comment (ukane-philemon) [reply client/webserver/locales/en-us.go:500]: Okay.

**Review by buck54321 — APPROVED** [review]

**Pull request merged**